sdk/lib/opentelemetry/sdk/trace/export/batch_span_processor.rb

```diff
@@ -123,7 +123,7 @@ def force_flush(timeout: nil) # rubocop:disable Metrics/MethodLength
               n = spans.size + snapshot.size - max_queue_size
               if n.positive?
                 snapshot.shift(n)
-                report_dropped_spans(n, reason: 'buffer-full')
+                report_dropped_spans(n, reason: 'buffer-full', context: "force_flush")
```

Comment:
```suggestion
                report_dropped_spans(n, reason: 'buffer-full', context: 'force_flush')
```
single quotes for consistency